app/models/album.rb

```diff
@@ -1,15 +1,15 @@
-# class Album < ActiveRecord::Base
-#   validates :title, presence: true, uniqueness: { scope: :artist }
-#   validates :artist, presence: true
-#   validates :genre, presence: true
-#   validates :length, presence: true, numericality: { only_integer: true, greater_than: 0 }
-#
-#
-# private
-#   def create_album
-#     c = Albums.new
-#     c.valid?
-#     c.save
-#   end
-#
-# end
+class Album < ActiveRecord::Base
+  validates :title, presence: true, uniqueness: { scope: :artist }
+  validates :artist, presence: true
+  validates :genre, presence: true
+  validates :length, presence: true, numericality: { only_integer: true, greater_than: 0 }
+
+
+private
+  def create_album
+    c = Albums.new
+    c.valid?
+    c.save
```

Comment:
Apart from this not actually creating anything ever (it never receives arguments), there's actually a method build in to Rails that does just what you're intending! It's called `create`
```ruby
Album.create(...args go here...)
```